src/core/stdc/errno.d

```diff
@@ -59,12 +59,12 @@ else version (FreeBSD)
         alias errno = __error;
     }
 }
-else version (linux)
+else version (CRuntime_Bionic)
```

Comment:
`linux` refers to kernel APIs, but I don't see errno declarations in anywhere other than libc headers.